lms/resources/_js_config/file_picker_config.py

```diff
@@ -8,7 +8,9 @@ class FilePickerConfig:
     def blackboard_config(cls, context, request, application_instance):
         """Get Blackboard files config."""
         files_enabled = application_instance.settings.get("blackboard", "files_enabled")
-        groups_enabled = context.blackboard_groups_enabled
+        groups_enabled = application_instance.settings.get(
```

Comment:
These replace canvas_groups_enabled and blackboard_groups_enabled.

They are only relevant in the filepicker.